vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -190,6 +189,26 @@ ReactInstanceWin::ReactInstanceWin(
           this,
           options.Properties,
           winrt::make<implementation::ReactNotificationService>(options.Notifications))} {
+  m_whenLoaded.AsFuture()
```

Comment:
nit: it feels like the indentation is not intuitivie.

Reply:
Agree, but we use the automated clang-format and I have no control over it.
I am adding some comments. Hopefully it will make it stand out a little bit easier.
See commit https://github.com/microsoft/react-native-windows/pull/7356/commits/ab9f063cff5dacfe0bbaec7d6efa167c6ab3c9eb

Reply:
Sorry, keep forgetting that...